src/topology/instances/add_circle.lean

```diff
@@ -54,6 +58,26 @@ section linear_ordered_field
 
 variables [linear_ordered_field 𝕜] [topological_space 𝕜] [order_topology 𝕜] (p q : 𝕜)
 
+lemma coe_nsmul {n : ℕ} {x : 𝕜} : (↑(n • x) : add_circle p) = n • (x : add_circle p) := rfl
+
+lemma coe_zsmul {n : ℤ} {x : 𝕜} : (↑(n • x) : add_circle p) = n • (x : add_circle p) := rfl
```

Comment:
Can you take advantage of #17048 now that it is merged? Otherwise these lemmas should be protected, but it would be cool to just not need them at all.
```suggestion
lemma coe_nsmul {n : ℕ} {x : 𝕜} : (↑(n • x) : add_circle p) = n • (x : add_circle p) := rfl

lemma coe_zsmul {n : ℤ} {x : 𝕜} : (↑(n • x) : add_circle p) = n • (x : add_circle p) := rfl
```

Reply:
Great idea, I can indeed. I have just dropped these and added `instance : coe_is_add_monoid_hom 𝕜 (add_circle p) :=` instead.